flow-typed/types/event-handler-option.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+declare type EventHandlerOption = {
```

Comment:
this name is extremely generic and will apply to anything which is an event, while it is a specific form for a specific handler.
```suggestion
declare type KeyboardEventResult = {
```

Reply:
we could use it in other cases like this, why do you want to rename it to specific existing use case?

Reply:
cause it is specific, where else will you have prevent default and payload response?
The response object is tightly coupled with the implementation of this class

Reply:
we didn't use, it could be used on the future..
it's use case I think.

Reply:
unless you see a concrete use case then it is specific.
refactor to generalize a use case later is always possible, but there is no benefit or use case here.
Let's change it to fit the part.

Reply:
OK